I/igraph/build_tarballs.jl

```diff
@@ -0,0 +1,80 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "igraph"
+version = v"0.10.15"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/igraph/igraph.git", "635b432eff0a89580ac9bb98068d2fbc8ef374f2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/igraph/
+
+
+if [[ "${target}" == *mingw* ]]; then
+    LBT=blastrampoline-5
+else
+    LBT=blastrampoline
+fi
+
+if [[ "${target}" == *apple* ]]; then
+    LTO=OFF
+else
+    LTO=ON
+fi
+
+
+CONF_FLAGS="\
+    -DBUILD_SHARED_LIBS=ON \
+    -DIGRAPH_ENABLE_LTO=${LTO} \
+	-DIGRAPH_ENABLE_TLS=ON \
+	-DIGRAPH_USE_INTERNAL_BLAS=OFF \
+	-DIGRAPH_USE_INTERNAL_LAPACK=OFF \
+	-DIGRAPH_USE_INTERNAL_ARPACK=OFF \
+	-DIGRAPH_USE_INTERNAL_GLPK=OFF \
+	-DIGRAPH_USE_INTERNAL_GMP=OFF \
+	-DIGRAPH_USE_INTERNAL_PLFIT=ON \
+	-DIGRAPH_GLPK_SUPPORT=ON \
+	-DIGRAPH_GRAPHML_SUPPORT=ON \
+	-DIGRAPH_OPENMP_SUPPORT=ON \
+    -DBLA_VENDOR=blastrampoline -DBLAS_LIBRARIES=\"${LBT}\" -DLAPACK_LIBRARIES=\"${LBT}\" \
+    -DBUILD_TESTING=OFF -DIGRAPH_WARNINGS_AS_ERRORS=OFF" # adapted from https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-igraph due to issues on Windows
```

Comment:
`BUILD_TESTING` is not recognized by igraph and will be ignored. Tests are not built by default, so this is not something to worry about.

Reply:
the windows build was failing for me without this -- I copied this from the msys2 repository

I will try again without it (given also your other comments about not using git)